evm_loader/program/src/account_storage.rs

```diff
@@ -40,6 +45,14 @@ impl<'a> ProgramAccountStorage<'a> {
     /// 1. contract code info
     /// 2. caller or caller account info(for ether account)
     /// 3. ... other accounts (with `clock_account` in any place)
+    /// 
+    /// # Errors
+    ///
+    /// Will return: 
+    /// `ProgramError::InvalidArgument` if account in `account_infos` is wrong or in wrong place
+    /// `ProgramError::InvalidAccountData` if account's data doesn't meet requirements
+    /// `ProgramError::NotEnoughAccountKeys` if `account_infos` doesn't meet expectations
+    #[allow(clippy::missing_panics_doc)]
```

Comment:
Why do you need `missing_panics_doc` here?

Reply:
Clippy's false positive panic triggered because of unwrap on line 165 - it could cause panic. But I've checked it already to be Some at line 155.

Reply:
I've made changes to resolve this situation and remove  allowance of `missing_panics_doc`